static/js/Router.tsx

```diff
@@ -1,25 +1,26 @@
 import React from "react"
-import { Route, Router as ReactRouter } from "react-router"
+import { BrowserRouter } from "react-router-dom"
 import { Provider } from "react-redux"
-import { History } from "history"
+import { Provider as ReduxQueryProvider } from "redux-query-react"
 import { Store } from "redux"
 
 import App from "./pages/App"
+import { getQueries } from "./store/configureStore"
```

Comment:
there's a `getQueries` function already set up in `lib/redux_query` which has the typing all set up already

Reply:
(`getEntities` too)